src/components/SkipButton/SkipButton.server.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import SkipButton from './SkipButton';
+
+describe(SkipButton, () => {
```

Comment:
Can we add any prefix/suffix to the name that indicates that it's a server or a client test please?